excude_files.txt

```diff
@@ -0,0 +1 @@
+.git*
```

Comment:
This filename is wrong, should be `exclude_`

Reply:
Made the changes